client/my-sites/plans/jetpack-plans/product-card/product-above-button-text.ts

```diff
@@ -14,7 +14,7 @@ export default function productAboveButtonText(
 		siteProduct &&
 		( JETPACK_SEARCH_PRODUCTS as ReadonlyArray< string > ).includes( product.productSlug )
 	) {
-		return translate( '*estimated price based off of %(records)s records', {
+		return translate( '*estimated price based on %(records)s records', {
```

Comment:
Not sure how I feel about including the asterisk in the translatable string. Is it necessary for supporting RTL languages?

Reply:
That's a tricky one. There's RTL to contend with and also potentially asterix not being the right character for a footnote in some languages. I will leave it in the translation for now.